src/Microsoft.Win32.SystemEvents/src/Configurations.props

```diff
@@ -4,11 +4,11 @@
     <PackageConfigurations>
       netstandard;
       netcoreapp2.0-Windows_NT;
```

Comment:
Do we need the netcoreapp2.0 configuration?

Reply:
netstandard is not supported assembly. Do we want to change it to netcoreapp2.1 instead? Since 2.0 is not LTS anymore?

Reply:
Isn't this a new package? Do we care about making it support anything beyond netcoreapp3.0?

Reply:
No, this is not a new package. This was released in 2.1 release. It is also part of the Compat pack which supports 2.0 so we can't drop 2.0 support unless we drop support from the compat pack and only support 2.1 and 3.0

Reply:
OK I missed it in my search as I think I searched for System.Win32. At any rate that would explain why it is here and should remain here for compatibility.